src/schemathesis/schemas.py

```diff
@@ -203,6 +217,10 @@ def parametrize(
         _code_sample_style = (
             CodeSampleStyle.from_str(code_sample_style) if isinstance(code_sample_style, str) else code_sample_style
         )
+        for name in ("method", "endpoint", "tag", "operation_id", "skip_deprecated_operations"):
+            value = locals()[name]
+            if value is not NOT_SET:
+                warn_filtration_arguments(name)
```

Comment:
Use existing filters, then replace / add new ones from these values. Then pass the result to clone

Reply:
In this case, clone will be simple